client/src/Components/Login/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Auth from './Auth';
+
+
+
+class LoginForm extends React.Component {
+
+
+	render() {
+		return (
+			<div>
+				<Auth />
+			</div>
```

Comment:
Może by przenieść to siedzi w Auth tutaj lub wywalić cały ten plik  a w Login wyrenderować Auth.